compiler+runtime/test/cpp/jank/runtime/obj/persistent_string.cpp

```diff
@@ -0,0 +1,70 @@
+/* clang-format off */
```

Comment:
Why is this here? None of the C++ files in the jank repo have formatting disabled.

Reply:
I used the one C++ file where that is not true as a starting point and assumed that was normal without checking.

https://github.com/jank-lang/jank/blob/676cd4f98a7514edb87839065a9719c1bdf362ef/compiler%2Bruntime/test/cpp/jank/native_persistent_string.cpp#L1

Reply:
Fair point. Can't fault you for following the precedent. Neither of these should have that set, at this point, though.